src/components/activity-list/index.js

```diff
@@ -1,2 +1,2 @@
 export { default as ActivityList } from './ActivityList';
-export { default as ActivityListHeader } from './ActivityListHeader';
+export { default as ActivityListHeader } from './FastActivityListHeader';
```

Comment:
Since the other one is removed, perhaps it makes sense to just leave the same name? Does it have to be named Fast?